library/Jira/MonitoringInfo.php

```diff
@@ -121,6 +122,19 @@ public function getStateName()
         return strtoupper($this->object->state->getStateText());
     }
 
+    public function getLastStateChange()
+    {
+        if ($this->object instanceof MonitoredObject) {
+            if ($this->object instanceof Service) {
+                return DateFormatter::formatDateTime($this->object->last_state_change);
+            }
+
+            return DateFormatter::formatDateTime($this->object->last_state_change);
```

Comment:
You access the same property in the same way in both cases. Why the distinction?